osu.Game/Modes/UI/HudOverlay.cs

```diff
@@ -32,6 +37,7 @@ public abstract class HudOverlay : Container
         protected HudOverlay()
         {
             RelativeSizeAxes = Axes.Both;
+            AlwaysPresent = true;

```

Comment:
rather than using this, make a content container which you then hide/show. leave the `HudOverlay` visible at all times (we shouldn't be adjusting our own `Alpha` from within whenever we can avoid it).